package/ios/RNCSliderManager.m

```diff
@@ -87,6 +87,16 @@ static void RNCSendSliderEvent(RNCSlider *sender, BOOL continuous, BOOL isSlidin
 {
   float value = [sender discreteValue:sender.value];
 
+  if (value < sender.lowerLimit) {
+      value = sender.lowerLimit;
+      [sender setValue:value animated:NO];
+  }
+    
+  if (value > sender.upperLimit) {
```

Comment:
Just like for Android - let's use `else if` here to avoid unnecessary calculations.